code/game/machinery/computer/communications.dm

```diff
@@ -414,8 +414,8 @@
 		menu_state=value
 
 /proc/call_shuttle_proc(mob/user, reason, sanitized = FALSE)
-	if(GLOB.sent_strike_team == 1)
-		to_chat(user, "<span class='warning'>Central Command will not allow the shuttle to be called. Consider all contracts terminated.</span>")
+	if(GLOB.strike_team_sent == 1)
+		to_chat(user, "<span class='warning'>Central Command will not allow the shuttle to be called. Please stand by.</span>") //This may show up before Epsilon Alert/Before DS arrives
```

Comment:
Very slightly more "official"-sounding (and also intentionally cryptic)
```suggestion
	if(GLOB.strike_team_sent)
		to_chat(user, "<span class='warning'>Central Command does not allow the shuttle to be called at this time. Please stand by.</span>") //This may show up before Epsilon Alert/Before DS arrives
```